Referenced/Referenced.csproj

```diff
@@ -1,7 +1,10 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <ItemGroup>
-    <ProjectReference Include="..\TransitiveReferenced\TransitiveReferenced.csproj" />
+    <ProjectReference Include="..\TransitiveReferenced\TransitiveReferenced.csproj">
+      <PrivateAssets>all</PrivateAssets>
+      <Private>False</Private>
+    </ProjectReference>
```

Comment:
I don't think this change makes sense in @rickstrahl's case, where "Referenced" is "The application that hosts some plugins", so you probably want _its_ output folder to be "complete".

Reply:
What I understand is that he not want the output folder complete.  The plugin references the application. But this application should not be in output directory. I had this use case as well in the past.

_Referenced_ is the plugin
_Referencing_ is the application for which the plugin is.

Reply:
> _Referenced_ is the plugin
> _Referencing_ is the application for which the plugin is.

I think that's backward; sorry for the confusing names I picked! Here Referencing is the plugin, and Referenced is the app. The plugin references the app (to get its `IAppPlugin` interfaces or whatever) but shouldn't redistribute it.

Reply:
Sorry my fault you are right:

Referencing is the plugin
Referenced is the application for which the plugin is.

Reply:
@trivalik Yes you have the use case correct:

* Addin references main application
* Addin output should **not** pull in all the application dependencies - should only reference the output assembly
  (ie. behave like a `<Reference>`)